src/Core/App.cs

```diff
@@ -26,13 +26,14 @@ public class App : Application
         private readonly IHost _host;
         public App()
         {
-            _host = Host.CreateDefaultBuilder()
+            _host = MobileBlazorBindingsHost.CreateDefaultBuilder()
                 .ConfigureServices((hostContext, services) =>
                 {
                     ConfigureServices(services);
                 })
                 .Build();
 
+            MainPage = new ContentPage();
```

Comment:
Just to make sure that I understand the subtlety of this change:

- Is this mandatory for every app? If yes, for what reasons?
- Can we instantiate something else than a `ContentPage`, or are there any plans for the future?

Reply:
This is a near-term workaround for a bug where the Blazor-based UI can sometimes take a bit too long to start up, and if it doesn't finish in time, the underlying platform will throw an error saying there's no UI to show, and your app crashes. Blazor is fundamentally asynchronous, which at this time isn't well-supported on the underlying platform. The workaround is to temporarily set an arbitrary `MainPage` that will get overridden asynchronously once the Blazor UI is done loading.

* Mandatory? Yes, sort of.
* Can you use something else? Yes, definitely. Any valid Page can be used.
* Future plans? Yes, in .NET MAUI, the successor to Xamarin.Forms, we will look at having a better way to do async startup.

Reply:
More info on this is here:

* https://github.com/xamarin/MobileBlazorBindings/issues/138
* https://github.com/xamarin/MobileBlazorBindings/pull/142

Reply:
Thanks again, but now I just want to drop everything and try that out! 🍻

Reply:
Oh please do! 😄 As always, we're super eager to get any feedback you have or help you out with working through issues. Mobile Blazor Bindings is an _experiment_ and we hope to learn as much as we can about what works, how does it feel, and where we can improve!